test/html-to-vdom/index.js

```diff
@@ -41,6 +41,19 @@ describe('htmlparser-to-vdom', function () {
             var converted = convertHTML(html);
             converted.properties.tabIndex.should.equal('1');
         });
+
+        it('parses an div with styles correctly', function () {
+
+            var html = '<div style="top: -7px; left: -6px; background: rgb(0,0,132);"></input>';
```

Comment:
Maybe some additional cases for missing spaces like `top:-7px;left:-6px;border:1px solid black;` might be useful.  
It also seems like browsers don't require a trailing semicolon eg: `style="top:-7px"` or `style="top: -7px; background:blue"`. I wonder if that should be supported as well.